content/docs/15.how-to-guides/long-running-intensive-tasks.md

```diff
@@ -0,0 +1,162 @@
+
+As an exmample, we will use a [dbt job](https://docs.getdbt.com/docs/running-a-dbt-project/run-your-dbt-projects) to demonstrate how Kestra can help you execute complex tasks on Kubernetes with resource requirements.
+
+## Schedule task in a Kubernetes pod using podCreate
+
+Kestra's [podCreate](https://kestra.io/plugins/plugin-kubernetes/tasks/io.kestra.plugin.kubernetes.podcreate) task allows you to launch a Kubernetes pod directly by providing the complete Kubernetes YAML configuration as input. This gives you full control over the pod’s specifications, such as CPU, memory, image, or node selector.
+
+Here is an example of a dbt job that runs on Kubernetes using Kestra:
+
+```yaml
+id: my-dbt-job
+namespace: dev
+
+tasks:
+  - id: dbt-command
+    type: io.kestra.plugin.kubernetes.PodCreate
+
+    # Retry the task if it fails
+    retry:
+      behavior: RETRY_FAILED_TASK
+      maxAttempt: 2
+      type: constant
+      interval: PT5M
+      warningOnRetry: true
+    namespace: kestra
+
+    # Define the commands to run
+    inputFiles:
+      dbt-commands.sh: |
+        #!/bin/bash
+
+        # Exit on error
+        set -eo pipefail
+
+        # Clone the dbt example repository
+        git clone --depth 1 https://github.com/dbt-labs/jaffle_shop_duckdb.git --branch duckdb --single-branch
+
+        # Copy the dbt example repository to the working directory
+        cp -a jaffle_shop_duckdb/. .
+
+        # dbt commands to run
+        dbt deps
+        dbt build
+
+    # Define the pod specification using the Kubernetes YAML syntax
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: dbt-duckdb
+          image: ghcr.io/kestra-io/dbt-duckdb:latest
+
+          # Specify resource requirements
+          resources:
+            request:
+              cpu: "300m"
+              memory: "500Mi"
+
+          # Run the script in the container
+          command:
+            - "/bin/bash"
+            - "{{workingDir}}/dbt-commands.sh"
+
+      # Node selector to run the pod on a specific node
+      nodeSelector: {}
+```
+
+This flow will:
+- create a Kubernetes pod in the `kestra` namespace with the specified resource requirements: 300m CPU and 500Mi memory
+- clone the dbt example repository inside the pod
+- run the dbt seed and build commands
+
+![dbt-pod-create](/docs/how-to-guides/kubernetes/pod_create_dbt.png)
+
+At the end of the execution, the pod is deleted, and the logs remains available in the Kestra UI.
+
+![dbt-pod-deleted-after-success](/docs/how-to-guides/kubernetes/pod_create_delete.png)
+
+## Embrace Kestra versality with Kubernetes Task Runners
+
+While podCreate provides deep control, it takes aways all the benefits of Kestra rich plugins ecosystem, [dbt plugin](https://kestra.io/plugins/plugin-dbt/tasks/cli/io.kestra.plugin.dbt.cli.dbtcli) in this case. Also it can be cumbersome to manage complex Kubernetes pod YAML specification for each task, especially when you have multiple commands to run.
```

Comment:
Change plugin link 

```suggestion
While podCreate provides deep control, it takes aways all the benefits of Kestra's rich plugins ecosystem, [dbt plugin](https://kestra.io/plugins/plugin-dbt/tasks/cli/io.kestra.plugin.dbt.cli.dbtcli) in this case. Also it can be cumbersome to manage complex Kubernetes pod YAML specification for each task, especially when you have multiple commands to run.
```